server.ts

```diff
@@ -260,6 +262,10 @@ restoreOverwrittenFilesWithOriginals().then(() => {
   app.use('/ftp(?!/quarantine)/:file', fileServer()) // vuln-code-snippet vuln-line directoryListingChallenge
   app.use('/ftp/quarantine/:file', quarantineServer()) // vuln-code-snippet neutral-line directoryListingChallenge
 
+  app.use('/.well-known/csaf/:folder/:file', wellKnownServer())
+  app.use('/.well-known/csaf/:file', wellKnownServer())
+  app.use('/.well-known', serveIndexMiddleware, serveIndex('.well-known', { icons: true, view: 'details' }))
+
```

Comment:
this is working for me. the order is important of the last one takes precedent over the earlier routes.

```suggestion
  app.use('/.well-known', serveIndexMiddleware, serveIndex('.well-known', { icons: true, view: 'details' }))
  app.use('/.well-known', express.static('.well-known'))

```